messenger.rst

```diff
@@ -19,6 +19,27 @@ install messenger before using it:
 
     $ composer require messenger
 
+Message
+-------
+
+Before you can send a message, you must create it first. There is no specific
+requirement for a message::
+
+    // src/Message/SendNotification.php
+    namespace App\Message;
+
+    class SendNotification
+    {
+        private $message;
```

Comment:
Could we change it to `$recipient`? As a beginner, I wouldn't know what the message is - is it the variable or the whole object?

Reply:
the idea is that we're sending a notification containing a `message`, not a message to a `recipient`.

take a look at the current dispatch example (https://symfony.com/doc/current/messenger.html#using-the-messenger-service). the sendnotification gets dispatched with a message.

Reply:
What about `content`? I agree `message` in a message might be misleading.

Reply:
@pamil i misunderstood your comment, sorry. while `$recipient` wouldn't fit the example, how about we change it to `$string` instead? we'll avoid confusing what a `message` is and basically tell that it can be whatever. what do you think?

Reply:
yeah, `content` should do it as well :)